packages/datetime/src/_datepicker.scss

```diff
@@ -109,6 +109,17 @@ $header-margin: ($header-height - $pt-input-height) / 2 !default;
     display: table-row;
   }
 
+  .DayPicker-WeekNumber {
+    display: table-cell;
+    cursor: pointer;
+    min-width: 1rem;
+    padding: 0.5rem;
+    vertical-align: middle;
+    text-align: right;
+    color: $gray3;
+    font-size: 0.75em;
```

Comment:
we don't use `em` or `rem` elsewhere in the codebase so these values will likely have to be refactored.

Reply:
```css.scss
width: $datepicker-day-size;
height: $datepicker-day-size;
font-size: $pt-font-size;

color: $pt-text-color-disabled;
// also for dark theme:
color: $pt-dark-text-color-disabled;
```